tests/Resources/TokenTest.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+namespace Plivo\Tests\Resources;
+
+
+
+
+use Plivo\Http\PlivoRequest;
+use Plivo\Http\PlivoResponse;
+use Plivo\Tests\BaseTestCase;
+
+
+/**
+ * Class TokenTest
+ * @package Plivo\Tests\Resources
+ */
+class TokenTest extends BaseTestCase
+{
+
+    function testTokenCreate()
+    {
+        $request = new PlivoRequest(
+            'POST',
+            'Account/MAXXXXXXXXXXXXXXXXXX/JWT/Token/',
+            [
+                'iss'=> "iss",
+                'sub'=> "kowshik"
+            ]);
+        $body = file_get_contents(__DIR__ . '/../Mocks/tokenCreateResponse.json');
```

Comment:
tokenCreateResponse.json file is not added

Reply:
Added